preprocessors/ner/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
+
+RUN apt-get update -y \
+        && apt-get install -y software-properties-common \
+        && add-apt-repository ppa:deadsnakes/ppa \
```

Comment:
What is this repository used to get?

Reply:
I'm using `FROM ubuntu:20.04` as my base container since it has java pre-installed. I'm using the repos to install python. Not sure it there's a better way.

Reply:
There is definitely a better way - Python is in the main ubuntu repos. You don't need to add another PPA for something like this. Python 3.x is available as the `python3` package since historically Python 2.x was available as `python`. Since 2 is past EoL, the `python` package no longer exists. In fact...you're probably installing python3 as a dependency of `python3-pip`. I'm not sure why you're installing it twice though (lines 9 and 11).